src/constants/vfx.asm

```diff
@@ -22,3 +22,6 @@ GROUND_VFX_NONE          equ $00
 GROUND_VFX_SHALLOW_WATER equ $05
 GROUND_VFX_GRASS         equ $06
 GROUND_VFX_PIT           equ $50
+
+; height of the window on the top of the screen
+WINDOW_HEIGHT equ 2 ; number in tiles
```

Comment:
`vfx.asm` is intended to hold constants for small visual effects rendered using sprites.
Wouldn't this constant be more in place in, say, `constants/gfx.asm` ?

Reply:
I did not understood the difference between vfx and gfx

Reply:
VFX is for Visual Effects, and is specifically around visuals produced using small sprites (especially around Link's feet, like water effects or dust smoke, but not only).

GFX is for Graphics (or Graphic Effetcs), and is everything else: graphic elements, backgrounds, etc.

So gfx.asm is the most generic constants file; while vfx.asm is very specific to one single purpose.